docs/source/entities-advanced.mdx

```diff
@@ -385,7 +540,7 @@ Because the router is already _ignoring_ `Bill.amount` in the Payments subgraph
 
 </Note>
 
-1. Remove the `@override` directive from the Billing subgraph, because it no longer has any effect:
+4. Remove the `@override` directive from the Billing subgraph, because it no longer has any effect:
```

Comment:
We need a step that "completes" the migration, i.e. `percent(100)` or remove the label to direct all traffic to the new subgraph. After that happens, then we can remove it from the old one.